src/Sulu/Bundle/TestBundle/Testing/TestUserProvider.php

```diff
@@ -141,11 +141,13 @@ public function refreshUser(UserInterface $user)
      */
     public function supportsClass($class)
     {
-        return $class === 'Sulu\Bundle\CoreBundle\Entity\TestUser';
+        return $class instanceof UserInterface;
```

Comment:
I wasn't able to Find a class named TestUser in sulu namespace, that's why I changed it. Could that be a relict of the past?

I can also change it back as it was. Not sure about it.